libraries/exoplayer/src/main/java/androidx/media3/exoplayer/drm/DefaultDrmSession.java

```diff
@@ -489,6 +492,9 @@ private long getLicenseDurationRemainingSec() {
 
   private void postKeyRequest(byte[] scope, int type, boolean allowRetry) {
     try {
+      if (type == ExoMediaDrm.KEY_TYPE_STREAMING) {
+        currentKeyLoadInfo = new KeyLoadInfo();
+      }
```

Comment:
This should probably just check for `mode != DefaultDrmSessionManager.MODE_RELEASE`.   That way we cover download requests too.

Reply:
Yes, that sounds like a good idea.  We can start a list for what to add to `KeyLoadInfo` here.

Reply:
This is done in the latest commit, except for adding the `KeyLoadInfo` to the `drmKeysRemoved()` method.   I'm fine with doing this too, however then we should rename `KeyLoadInfo` to `KeyRequestInfo`.  I'm all for the [YAGNI](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it) thinking too ;-)   

But, seems most of the work will be rebasing all the files open for this change so it is probably now or never.

I'll follow your lead on it.

Reply:
> I'm fine with doing this too, however then we should rename `KeyLoadInfo` to `KeyRequestInfo`

Yeah, I think we should do this rename and include key releasing - I agree it's going to be easier to do this now than later.